ui/src/pages/Login.js

```diff
@@ -14,8 +14,8 @@ function Login() {
     e.preventDefault();
     setAuthAttempted(true);
     try {
-      const { data: auth } = await api.login(email.value, password.value);
-      dispatch({ type: 'LOGIN', auth });
+      await api.login(email.value, password.value);
```

Comment:
Ideally a 401 from the API would erase cookies like the logout endpoint does, but I don't know how to intercept the error response in flask